<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>713</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>curl</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>7221436_reasonably_small.gif</normaloff>7221436_reasonably_small.gif</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="Type">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>94</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="currentText">
     <string/>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="URL">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>881</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="placeholderText">
     <string>URL</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="RequestHeaders">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>981</width>
      <height>71</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Headers</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="RequestBody">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>981</width>
      <height>71</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Body</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Send">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>981</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="ResponseBody">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>981</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(200, 200, 190);</string>
    </property>
    <property name="placeholderText">
     <string>Body</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="ResponseHeaders">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>981</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(200, 200, 190);</string>
    </property>
    <property name="placeholderText">
     <string>Headers</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
